frontend/src/core/api/types.js

```diff
@@ -43,14 +43,26 @@ export type Entities = Array<Entity>;
 
 
 /*
- * Translation of an entity in a locale other than the currently selected.
+ * Preferred and Other Translations of an entity in a locale other than
+ * the currently selected locale.
+ */
+export type OtherLocaleTranslations = {|
+    +preferred: Array<OtherLocaleTranslation>,
+    +other: Array<OtherLocaleTranslation>,
+|};
+
+
+/*
+ * Translation of an entity in a locale.
```

Comment:
Nit: "Translation of an entity to a locale other than the currently selected locale."